hydro_lang/Cargo.toml

```diff
@@ -41,6 +43,10 @@ tokio = { version = "1.29.0", features = [ "full" ] }
 toml = { version = "0.8.0", optional = true }
 trybuild-internals-api = { version = "1.0.99", optional = true }
 
+[target.'cfg(target_os = "linux")'.dependencies]
+chrono = "0.4.39"
```

Comment:
We should make these deps optional and only enable when we want runtime measure.